{"name":"To verify user not able to SignIn with invalid credentials","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Try to login with invalid email/phonenumber expected [false] but found [true]","trace":"java.lang.AssertionError: Try to login with invalid email/phonenumber expected [false] but found [true]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertFalse(Assert.java:67)\r\n\tat pageTestCases.LoginTest.invalidSignIn(LoginTest.java:90)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(ArrayList.java:1257)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)\r\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\r\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)\r\n"},"stage":"finished","description":"Test Case Description:To verify user not able to SignIn with Invalid Credentials","steps":[{"name":"Method to open Signin form","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620734120541,"stop":1620734120548},{"name":"Method to enter email or mobile no","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620734122508,"stop":1620734122508},{"name":"Method to click on Continue","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620734122649,"stop":1620734122649},{"name":"Method to show invalid email response","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620734124121,"stop":1620734124121}],"attachments":[{"name":"Page Screenshot","source":"f55a6924-ca14-486d-8026-0e3efc7772f1-attachment","type":"image/png"},{"name":"invalidSignInFailed and screenshot Taken!","source":"10d1adfd-8cbf-423f-9937-dc4f1959836b-attachment","type":"text/plain"}],"parameters":[],"start":1620734114302,"stop":1620734124497,"uuid":"a00d1bbe-7349-4803-a573-9dc34e77dc65","historyId":"af4eb947f79d6d11833e155ce3d35b98","fullName":"pageTestCases.LoginTest.invalidSignIn","labels":[{"name":"package","value":"pageTestCases.LoginTest"},{"name":"testClass","value":"pageTestCases.LoginTest"},{"name":"testMethod","value":"invalidSignIn"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"pageTestCases.LoginTest"},{"name":"host","value":"PC"},{"name":"thread","value":"4748@PC.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Story Name: To check user not able to SignIn with invalid credentials"},{"name":"severity","value":"critical"}],"links":[]}